sys/src/lib.rs

```diff
@@ -31,16 +34,25 @@ extern "C" {
     pub fn sha256(value_len: u64, value_ptr: u64, register_id: u64);
     pub fn keccak256(value_len: u64, value_ptr: u64, register_id: u64);
     pub fn keccak512(value_len: u64, value_ptr: u64, register_id: u64);
+    pub fn ripemd160(value_len: u64, value_ptr: u64, register_id: u64);
+    pub fn ecrecover(
```

Comment:
I wonder if we can't just link the docs from nearcore directly into these sys calls. I feel like it would be a painpoint eventually for someone who's trying to optimize their contracts but not directly know what all these syscalls are, and have to indirectly discover [near-vm-logic/logic.rs](https://github.com/near/nearcore/blob/master/runtime/near-vm-logic/src/logic.rs) to understand them